### PR TITLE
feat: add trading page filter for market type (isolated/cross/all)

### PR DESCRIPTION
**Pull request by tyleroooo**
image

### CODE REVIEW

**Comment by tyleroooo** [comment]
Doesn't work for Fills yet because subaccount fills doesn't have info about the relevant subaccount number

File: src/pages/trade/HorizontalPanel.tsx [hunk 1]
```diff
@@ -276,28 +285,54 @@ export const HorizontalPanel = ({ isOpen = true, setIsOpen }: ElementProps) => {
...
+            <$SelectMenu
+              value={viewIsolated}
+              onValueChange={(newViewIsolated: string) => {
+                setViewIsolated(newViewIsolated as MarketTypeFilter);
+              }}
+            >
+              <$SelectItem
+                key={MarketTypeFilter.AllMarkets}
+                value={MarketTypeFilter.AllMarkets}
+                label="All"
```
Comment (jaredvu) [inline src/pages/trade/HorizontalPanel.tsx:321]: use stringGetter

File: src/pages/trade/HorizontalPanel.tsx [hunk 2]
```diff
@@ -276,28 +285,54 @@ export const HorizontalPanel = ({ isOpen = true, setIsOpen }: ElementProps) => {
...
+              />
+              <$SelectItem
+                key={MarketTypeFilter.Isolated}
+                value={MarketTypeFilter.Isolated}
+                label="Isolated"
+              />
+              <$SelectItem
+                key={MarketTypeFilter.Cross}
+                value={MarketTypeFilter.Cross}
+                label="Cross"
```
Comment (jaredvu) [inline src/pages/trade/HorizontalPanel.tsx:331]: STRING_KEYS.CROSS

File: src/pages/trade/HorizontalPanel.tsx [hunk 3]
```diff
@@ -306,6 +341,10 @@ export const HorizontalPanel = ({ isOpen = true, setIsOpen }: ElementProps) => {
   );
 };
 
+const $VerticalSeparator = styled(VerticalSeparator)`
+  height: 1em !important;
```
Comment (jaredvu) [inline src/pages/trade/HorizontalPanel.tsx:345]: Instead of using !important, can we try using
```
&& {
}
```

**Review by jaredvu — APPROVED** [review]

**Pull request merged**